exercises/all-your-base/example.rb

```diff
@@ -1,17 +1,22 @@
 module BookKeeping
-  VERSION = 1
+  VERSION = 2
```

Comment:
Bumped test version since we now expect invalid base / input to raise an error rather than return nil